File: it.unibo.bls19Local/src/main/java/it/unibo/bls/devices/gui/segm7/LedSegmentHorizontal.java
package it.unibo.bls.devices.gui.segm7;

public class LedSegmentHorizontal extends LedSegment{

    public LedSegmentHorizontal(String name, int width, int height ){
        super(name, width, height);
    }

    @Override
    protected void setLedRep(){
        ledRep.addPoint(x+10,y+8);
        ledRep.addPoint(x+100,y+8);
        ledRep.addPoint(x+110,y+15);
        ledRep.addPoint(x+100,y+22);
        ledRep.addPoint(x+10,y+22);
        ledRep.addPoint(x+ 2,y+15);
    }
}

File: it.unibo.bls19Local/src/main/kotlin/it/unibo/chain/segment7/LedSegmentHorizontal.kt
package it.unibo.chain.segment7

class LedSegmentHorizontal(name: String, width: Int, height: Int) : LedSegment(name, width, height) {

    override fun setLedRep() {
        ledRep.addPoint(x + 10, y + 8)
        ledRep.addPoint(x + 100, y + 8)
        ledRep.addPoint(x + 110, y + 15)
        ledRep.addPoint(x + 100, y + 22)
        ledRep.addPoint(x + 10, y + 22)
        ledRep.addPoint(x + 2, y + 15)
    }
}
